### PR TITLE
Move contact page to Reference

### LINKED ISSUE #4615: `Enterprise Support` statement seems to be a bit misplaced
## Summary

`Enterprise Support` statement seems to be a bit misplaced

## How to reproduce

1. Search for `Enterprise Support`.
2. Get to this page (first result) https://docs.camunda.io/docs/next/components/zeebe/zeebe-overview/#enterprise-support-for-zeebe

=> It seems a bit off that we describe "Enterprise Support" specifically for Zeebe. Given that we offer Zeebe now together with Operate, Tasklist, Identity, ... as Camunda 8 under the [Camunda License version 1.0](https://github.com/camunda/camunda?tab=readme-ov-file#license) if would expect that we document "Enterprise Support" availability more centrally for all components.

## Suggestion for improvement

* Remove https://docs.camunda.io/docs/next/components/zeebe/zeebe-overview/#enterprise-support-for-zeebe
* Add overaching "Enterprise Support" page in https://docs.camunda.io/docs/next/reference/ - link to https://camunda.com/services/enterprise-support-guide/.
  * also see https://camunda.slack.com/archives/C0PLU0Y12/p1731915584941029

### PR DESCRIPTION
**Pull request by akeller**
## Description

Closes #4615. 

This will still be in the footer, which is a little wonky as it will always link to the current docs.

I did not adjust any of the metadata because the term (enterprise support) is already included.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [x] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by akeller** [comment]
Removing reviews, need to fix links, remove the section on the Zeebe page, and add redirects!

**Comment by akeller** [comment]
I added a single redirect that routes to the current/unversioned docs. The contact page (when it lived in the footer) wasn't versioned. Moving it to the Reference section with the rest of the docs content will cause it to be versioned.

`RewriteRule ^contact/?$ /docs/reference/contact/$1 [R=301,L]`

File: docs/reference/contact.md [hunk 1]
```diff
@@ -16,19 +17,17 @@ keywords:
...
 
-- For **security-related issues**, review our [security notices](../docs/reference/notices) for the most up-to-date information on known issues and steps to report a vulnerability so we can solve the problem as quickly as possible. Do not use GitHub for security-related issues.
+- For **security-related issues**, review our [security notices](/reference/notices.md) for the most up-to-date information on known issues and steps to report a vulnerability so we can solve the problem as quickly as possible. Do not use GitHub for security-related issues.
 
 - **Feedback and support** can be submitted or requested via JIRA by following our [Enterprise support process](https://camunda.com/services/enterprise-support-guide/). All users can also find feedback and support options in the Help Center or [Camunda community forum](https://forum.camunda.io/).
 
-- For sales inquiries, information about Camunda 8 performance and benchmarking, or anything not listed above, use our [Contact Us](https://camunda.com/contact/) form.
+- For sales inquiries, information about Camunda 8 performance and benchmarking, or anything not listed above, use our [Contact Us](/reference/contact.md) form.
```
Comment (conceptualshark) [inline docs/reference/contact.md:30]: I think this one should still point to the https://camunda.com/contact/ form?

**Review by conceptualshark — COMMENTED** [review]
I left one comment on the actual Contact page, but the updates look good to me otherwise!

**Review by pepopowitz — DISMISSED** [review]
One small suggestion for a change, but looks good to me!

**Review by conceptualshark — APPROVED** [review]

**Pull request merged**